bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/StrandResourceCallback.java

```diff
@@ -35,7 +35,7 @@
     private TransactionLocalContext transactionLocalContext;
 
     StrandResourceCallback(BType retType, CallableUnitCallback resourceCallback, ChannelDetails[] sendIns) {
-        super(retType, sendIns);
+        super(retType, sendIns, null);
```

Comment:
We have passed null as the third parameter in 2 instances "StrandResourceCallback" and "StrandWaitCallback", instead of passing null in the constructor, shall we have another constructor in "StrandCallback" which takes in only 2 arguments?